src/context/schemas/organization.schema.json

```diff
@@ -1,18 +1,18 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "https://fdc3.finos.org/schemas/1.0/organization.schema",
+  "$id": "https://fdc3.finos.org/schemas/next/organization.schema",
   "type": "object",
   "allOf": [{ "$ref": "context.schema#/" }],
   "properties": {
-    "type": { "enum": ["fdc3.organization"] },
+    "type": { "const": "fdc3.organization" },
     "id": {
       "type": "object",
       "properties": {
         "LEI": { "type": "string" },
         "PERMID": { "type": "string" },
-        "FDSID": { "type": "string" }
+        "FDS_ID": { "type": "string" }
       }
     }
   },
-  "required": ["id","name"]
+  "required": ["id"]
```

Comment:
I thought we decided `name` was required for organization, but according to https://fo.finos.org/docs/objects/organization it is optional?

Reply:
it seems more consistent for just id to be required